### PR TITLE
SIP-360 Update

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: content/sips/sip-360.md [hunk 1]
```diff
@@ -50,23 +50,23 @@ The representation of election winners as NFTs allows for direct integrations wi
...
 
-- **Election Period** - At the beginning of this period, anyone can call the `commence` function on the mothership. This function uses **Pyth Entropy** to select a random timestamp from the previous period to use for voting power calculation. Throughout the voting period, users may cast a ballot on the mothership, distributing their voting power across an array of nominees. Voting power is determined with **Wormhole Queries**, which is used to iterate over an array of snapshot contracts on specified chains to read values at the selected timestamp.
+- **Election Period** - At the beginning of this period, anyone can call the `commence` function on the mothership. This finalizes the list of candidates, sending them to the satellites via a cross-chain message. Throughout the election period, users on any chain with a mothership or satellite deployment can cast their votes, distributing their voting power across an array of candidates.
```
[the remark this answers is not in the document]
Comment (noahlitvin) [reply content/sips/sip-360.md:55]: I was thinking:
* We have the mothership incentivizing function calls that trigger a cross-chain message to capture snapshots everywhere throughout the administration period.
* On `commence`, the mothership sends a random [timepoint](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/utils/Votes.sol#L89) from the previous period and a finalized list of candidates.
* Throughout the election period, votes are recorded on each chain (no cross-chain messaging)
* On `resolve`, cross-chain queries tally the total votes to determine the winner (and sends a final cross-chain message to move the NFTs)

I can update the SIP to make this clearer if this makes sense?

File: content/sips/sip-360.md [hunk 2]
```diff
@@ -50,23 +50,23 @@ The representation of election winners as NFTs allows for direct integrations wi
...
 
-The use of timestamp randomness mitigates the risk of a governance attack in which a user flash loans assets and deposits them on the block where the voting power is calculated.
+This is system is fully permissionless. There is no need for specific addresses to have elevated permissions, the creation of merkle trees, or reliance on keepers. Election winners will be motivated to call the `resolve` function as soon as possible, such that they receive NFTs. The`commence` function can be triggered automatically when the first vote is cast in the election period, if necessary.
```
[the remark this answers is not in the document]
Comment (noahlitvin) [reply content/sips/sip-360.md]: Pushed an update to clarify.

File: content/sips/sip-360.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (noahlitvin) [reply content/sips/sip-360.md:55]: Yea good call, nomination is a cross-chain message. Will update 🙏

**Review by barrasso — APPROVED** [review]

**Comment by kaleb-keny** [comment]
should i merge this @barrasso @noahlitvin ?

**Pull request merged**